### DISCUSSION TITLE
Unable to retrieve ERA5-Land daily precipitation and ET₀

### DISCUSSION THREAD

**Discussion by kfujisaki** [open-meteo/open-meteo#825] [Q&A] [answered]
Hi,
I am using the API with R to collect historical data of daily precipitation and ET₀ in several locations in Europe.
I guessed that with default parameters, the data source would be ERA5-Land, but it seems that the data come from ERA5.
Here is two examples with the API query by url (location is Paris, France) : there is no result for ERA5-Land :
https://archive-api.open-meteo.com/v1/archive?latitude=48.8534&longitude=2.3488&start_date=2000-01-01&end_date=2000-01-31&daily=precipitation_sum,et0_fao_evapotranspiration&timezone=auto&models=best_match

https://archive-api.open-meteo.com/v1/archive?latitude=48.8534&longitude=2.3488&start_date=2000-01-01&end_date=2000-01-31&daily=precipitation_sum,et0_fao_evapotranspiration&timezone=auto&models=era5_land

What is the explanation ? IIRC, ERA5-Land includes daily precipitation and variables allowing the calculation of ET0 ? Is there somewhere a list of variables from ERA5-Land provided by the openmeteo API ?
Thanks !
Kenji

**Comment by patrick-zippenfenig** [comment]
ET0 also depends on short wave radiation and wind speed which is not available from era5-land.

If you select `era5_seamless` it takes temperature and humidity from ERA5-land and wind speed + radiation from ERA5.

**Comment by guidocioni** [comment, marked as answer]
> Hi, I am using the API with R to collect historical data of daily precipitation and ET₀ in several locations in Europe. I guessed that with default parameters, the data source would be ERA5-Land, but it seems that the data come from ERA5. Here is two examples with the API query by url (location is Paris, France) : there is no result for ERA5-Land : https://archive-api.open-meteo.com/v1/archive?latitude=48.8534&longitude=2.3488&start_date=2000-01-01&end_date=2000-01-31&daily=precipitation_sum,et0_fao_evapotranspiration&timezone=auto&models=best_match
> 
> https://archive-api.open-meteo.com/v1/archive?latitude=48.8534&longitude=2.3488&start_date=2000-01-01&end_date=2000-01-31&daily=precipitation_sum,et0_fao_evapotranspiration&timezone=auto&models=era5_land
> 
> What is the explanation ? IIRC, ERA5-Land includes daily precipitation and variables allowing the calculation of ET0 ? Is there somewhere a list of variables from ERA5-Land provided by the openmeteo API ? Thanks ! Kenji

for the explanation on why precipitation is missing have a look at this past issue https://github.com/open-meteo/open-meteo/issues/453

**Comment by kfujisaki** [comment]
Thanks Patrick and Guido for your explanations :)
